### DISCUSSION TITLE
How to skip transactions

### DISCUSSION THREAD

**Discussion by miroslavvojtus** [flamingock/mongock#57] [Q&A] [answered]
Hi Mongock team,

we are facing similar issue as in https://stackoverflow.com/questions/67076023/mongock-migration-failure-with-transaction

In our change set we are trying to create indexes what is not allowed in mongodb transactions (neither listing and creating of collections).

Is there a way how to take out a change set or log from the transaction?
Currently we had to disable transactions completely.
We would like to have them for real data adaptions which would benefit from transactions.

THX
Miroslav

**Comment by dieppa** [comment, marked as answer]
Hello @miroslavvojtus ,

luckily we are a PR raised this morning for that, so it should be merged soon, probably tomorrow. However, as said in the ticket, we are not adding any new feature tu version 4, so this will be available in version 5, which we expect to have released by the end of June. However, we are delivering beta versions for those clients that want to use it meanwhile or just to check.

I can commit to have a beta version released on Friday as latest. Does this work for you?

Regards
